apps/explorer/src/ui/TransactionAddressSection.tsx

```diff
@@ -58,8 +58,11 @@ export function RecipientTransactionAddresses({
             title={recipients.length > 1 ? 'Recipients' : 'Recipient'}
         >
             <div className="flex flex-col gap-4">
-                {recipients.map(({ address, amount, coinType }) => (
-                    <div className="flex flex-col gap-0.5" key={address}>
+                {recipients.map(({ address, amount, coinType }, i) => (
```

Comment:
Not applicable to current PR, but we should consider aggregate the amounts for the same recipient. cc @Jordan-Mysten